app/model/IDatabaseWrapper.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Michal Struna
+ * Date: 24/11/2018
+ * Time: 16:16
+ */
+
+namespace App\Model;
+
+interface IDatabaseWrapper {
+
+    /**
+     * Select items from DB.
+     * @return array Array of DB items.
+     */
+    public function getAll(): array; // TODO: Array of selected columns in parameter?
```

Comment:
Tady by to chtělo přidat pole sloupců do selectu, abychom při získávání všech sloupců z DB netahali i všechny sloupce. Problém je v tom, že PDO nepodporuje prepared statements pro názvy sloupců.